scripts/zones/Balgas_Dais/mobs/gilagoge_tlugvi.lua

```diff
@@ -0,0 +1,34 @@
+-----------------------------------
+-- Area: Balga's Dais
+--  Mob: Gilagoge Tlugvi
+-- KSNM: Seasons Greetings
+-----------------------------------
+mixins = {require("scripts/mixins/job_special")}
+require("scripts/globals/status")
+-----------------------------------
+local entity = {}
+
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+end
+
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.SUPERLINK, 0)
+    mob:setMobMod(xi.mobMod.NO_LINK, 1)
+end
+
+entity.onAdditionalEffect = function(mob, target, damage)
+    -- en-dispel not implemented
+    -- return xi.mob.onAddEffect(mob, target, damage, xi.mob.ae.DISPEL)
```

Comment:
Since this is needed for two BCNM/KSNMs can we try implementing this before we merge this?

Reply:
I'll hop on it :)